randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -81,142 +105,245 @@ Extra - asset_id: collision_camera_001
+  * Power Beam Door to collision_camera_001 (G)/Door to Robot Fight Arena
   * Extra - actor_name: doorpowerpower_000
   * Extra - actor_def: actordef:actors/props/doorpowerpower/charclasses/doorpowerpower.bmsad
   * Extra - left_shield_entity: {EMPTY}
   * Extra - left_shield_def: None
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
+  > Door to collision_camera_003 (G)
+      Trivial
 
 > Door to collision_camera_003 (G); Heals? False
-  * Power Beam Door to collision_camera_003 (G)/Door to collision_camera_002 (G)
+  * Power Beam Door to collision_camera_003 (G)/Door to Robot Fight Arena
   * Extra - actor_name: doorpowerpower_017
   * Extra - actor_def: actordef:actors/props/doorpowerpower/charclasses/doorpowerpower.bmsad
   * Extra - left_shield_entity: {EMPTY}
   * Extra - left_shield_def: None
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
+  > Door to collision_camera_001 (G)
+      Trivial
 
 > Start Point; Heals? False; Spawn Point
   * Extra - start_point_actor_name: SP_Checkpoint_Dead_ChozoRobotSoldier
   * Extra - start_point_actor_def: actordef:actors/logic/startpoint/charclasses/startpoint.bmsad
+  > Door to collision_camera_003 (G)
+      Trivial
 
 ----------------
 collision_camera_003 (G)
 Extra - total_boundings: {'x1': -15400.0, 'x2': -10900.0, 'y1': -6000.0, 'y2': -700.0}
 Extra - polygon: [[-10900.0, -700.0], [-13000.0, -700.0], [-13000.0, -3700.0], [-15400.0, -3700.0], [-15400.0, -6000.0], [-10900.0, -6000.0]]
 Extra - asset_id: collision_camera_003
-> Dock to collision_camera_004 (G); Heals? False
-  * EMMI Door to collision_camera_004 (G)/Dock to collision_camera_003 (G)
+> Dock to Blue EMMI Introduction; Heals? False
+  * EMMI Door to Blue EMMI Introduction/Dock to collision_camera_003 (G)
   * Extra - actor_name: dooremmy_000
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
+  > Floating Platform
+      Trivial
 
-> Dock to Transport to Burenia; Heals? False
-  * Not Determined to Transport to Burenia/Dock to collision_camera_003 (G)
+> Door to Transport to Burenia; Heals? False
+  * Plasma Beam Door to Transport to Burenia/Door to collision_camera_003 (G)
   * Extra - actor_name: doorpowerpower_018
   * Extra - actor_def: actordef:actors/props/doorpowerpower/charclasses/doorpowerpower.bmsad
   * Extra - left_shield_entity: Root:pScenario:rEntitiesLayer:dctSublayers:default:dctActors:door_shield_plasma_000
   * Extra - left_shield_def: actordef:actors/props/door_shield_plasma/charclasses/door_shield_plasma.bmsad
   * Extra - right_shield_entity: Root:pScenario:rEntitiesLayer:dctSublayers:default:dctActors:door_shield_plasma_002
   * Extra - right_shield_def: actordef:actors/props/door_shield_plasma/charclasses/door_shield_plasma.bmsad
-
-> Door to collision_camera_002 (G); Heals? False
-  * Power Beam Door to collision_camera_002 (G)/Door to collision_camera_003 (G)
+  > Ledge Grab
```

Comment:
There's a missing trivial connection to the Door to Transport to Burenia here.

Reply:
done